inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Could s_dev be rewritten to standard_deviation to avoid confusion?

Reply:
Will do.